sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -957,9 +957,9 @@ public partial class MetricAnomalyAlertScope
     public partial class MetricAnomalyAlertSnoozeCondition
     {
         public MetricAnomalyAlertSnoozeCondition(int autoSnooze, Azure.AI.MetricsAdvisor.Models.SnoozeScope snoozeScope, bool isOnlyForSuccessive) { }
-        public int AutoSnooze { get { throw null; } }
-        public bool IsOnlyForSuccessive { get { throw null; } }
-        public Azure.AI.MetricsAdvisor.Models.SnoozeScope SnoozeScope { get { throw null; } }
+        public int AutoSnooze { get { throw null; } set { } }
+        public bool IsOnlyForSuccessive { get { throw null; } set { } }
+        public Azure.AI.MetricsAdvisor.Models.SnoozeScope SnoozeScope { get { throw null; } set { } }
```

Comment:
Adding setters to some properties to make it easier for users to update them (instead of having to create a whole new object). They're not nullable and we do no validation on them, so I believe making them settable is convenient and won't do harm. They're still required in the constructor.

Reply:
This is all because this is an In-Out object right?
If so it seems consistent with the behavior for MA. I still find it weird :P

Reply:
> This is all because this is an In-Out object right?

Correct.